src/librustc/hir/def.rs

```diff
@@ -46,7 +46,7 @@ pub enum NonMacroAttrKind {
 pub enum Def {
     // Type namespace
     Mod(DefId),
-    Struct(DefId), // DefId refers to NodeId of the struct itself
+    Struct(DefId), // `DefId` refers to `NodeId` of the struct itself
```

Comment:
Nit: It would be good to move cosmetic and whitespace changes into a separate commit, if there are many of them.

Reply:
Sorry, you're right!

Reply:
Done this now by interactively rebasing.